.github/workflows/catpowder.yml

```diff
@@ -46,6 +46,24 @@ jobs:
         echo -e "\tPasswordAuthentication no" >> $HOME/.ssh/config
         echo -e "\tUser ${{ secrets.USERNAME }}" >> $HOME/.ssh/config
         echo -e "\tPort ${{ secrets.PORTNUM }}" >> $HOME/.ssh/config
+    - name: Bootstrap VMs
+      shell: bash
+      run: |
+        echo "Running iptables rule on SERVER"
+        ssh $SERVER "sudo iptables -C OUTPUT -p tcp -o eth1 --tcp-flags RST RST -j DROP || \
+                     sudo iptables -A OUTPUT -p tcp -o eth1 --tcp-flags RST RST -j DROP"
```

Comment:
It is not guaranteed (nor safe) to assume that  `eth1` is the network interface that we should target (this may change on reboot). A safer automated proceedure would query the target interface based on the IP address (which we can get from our secrets).

Reply:
Refer to other duplicate comment for details.